src/taskpane/pages/dashboard.tsx

```diff
@@ -74,11 +111,37 @@ function Dashboard({ citeSupport }: DashboardProps): ReactElement {
     });
   };
 
-  const unCheckAllCheckboxes = () => {
-    setItems((currentItems) => {
-      return currentItems.map(unCheckCheckbox);
+  async function getSelectedCitation(): Promise<void> {
```

Comment:
Would this work (with an empty dependency array at the end)? Then you should be able to add `getSelectedCitation` to the dependency array further down. I think it should retain the correct behavior

```suggestion
    const getSelectedCitation = useCallback(async (): Promise<void> => {
```